rpa-manager/modules/ROOT/pages/processoperations-deploymentmap.adoc

```diff
@@ -1,11 +1,19 @@
 = Deployment Map
 
-The deployment map shows you the current status of each productive configuration of all processes for which you are a team member.
+The deployment map shows the current status of each run configuration of all processes for which you are a team member.
 
-The interrelationships of processes, RPA Bots and Configurations are mapped by a network.
+The interrelationships of processes, RPA Bots, and configurations are mapped as a network.
 
+You can view and manage all your run configurations in one place:
+
+* Switch between test and production configurations.
+* Switch between hierarchical and star network views.
+* Filter the bots and configurations by status.
 * Switch directly to the detailed view of a process or a configuration.
-* Execute, pause, and stop configurations directly from here.
+* Execute test configurations directly from here.
+* Execute, pause, and stop production configurations directly from here.
+
+The deployment map does not show invokable run configurations that were deleted in RPA Manager. Invokable run configurations deleted in RPA Manager still exist in Anypoint Exchange, but cannot be run by the bot. The deployment map might still show invokable configuration assets deprecated in Anypoint Exchange. Make sure to sync configuration changes manually between RPA Manager and Anypoint Exchange.
```

Comment:
In this one, the main action is to manually keep the configuration changes in sync. I suggest making a change like the following to make this active voice, present tense, and more clear, readable, and parallel in structure.

You must manually keep configuration changes done in RPA Manager and Anypoint Exchange in sync. For example:

. Invokable run configurations deleted in RPA Manager might still exist in Anypoint Exchange, but they cannot cannot be run by the bot and do not appear in RPA Manager. 
. Invokable configuration assets deprecated in Anypoint Exchange might still appear in RPA Manager.